src/compositemark/errorband.ts

```diff
@@ -57,9 +57,9 @@ export interface ErrorBandConfig extends ErrorBandPartsMixins {
    * The line interpolation method for the error band. One of the following:
    * - `"linear"`: piecewise linear segments, as in a polyline.
    * - `"linear-closed"`: close the linear segments to form a polygon.
-   * - `"step"`: alternate between horizontal and vertical segments, as in a step function.
-   * - `"step-before"`: alternate between vertical and horizontal segments, as in a step function.
-   * - `"step-after"`: alternate between horizontal and vertical segments, as in a step function.
+   * - `"step"`: produces a piecewise constant function (a step function) consisting of alternating horizontal and vertical lines. The y-value changes at the midpoint of each pair of adjacent x-values.
+   * - `"step-before"`: produces a piecewise constant function (a step function) consisting of alternating horizontal and vertical lines. The y-value changes before the x-value.
+   * - `"step-after"`: produces a piecewise constant function (a step function) consisting of alternating horizontal and vertical lines. The y-value changes after the x-value.
```

Comment:
Now we have sentences that start with a lowercase letter and are still inconsistent (only the new sentences start with "produces".

Reply:
means I have to write "produces" as "Produces"?

Reply:
Yes. Otherwise it's awkward.

Reply:
Thanks, @domoritz now you can check.